tasks/mochaTest.js

```diff
@@ -1,25 +1,47 @@
+const path = require('path');
+
+const defaultOptions = {
+  // Timeout is long due to tests that check *all* rules have been run.
+  timeout: 15000,
```

Comment:
This is the only bit that seems worse than before; I presume for my patch we could just up the one slow test's timeout? Or no?